vimrc

```diff
@@ -165,9 +165,10 @@ map <Leader>i mmgg=G`m<CR>
 syntax on
 set t_Co=256
 set background=dark
-colorscheme solarized
+colorscheme smyck
```

Comment:
Interesting conversation.  Do you guys think there is value in everyone having the same color scheme from a consistency standpoint? Or is this the type of thing where we can each introduce a little personality to our own computers?